### PR TITLE
fix: changelog: Fix `files.length` conditions

### PR DESCRIPTION
**Pull request by Turbo87**
After using `shift()` in the line above, the array is one item shorter than before. So to detect if there are multiple candidates we need to check if there are any items left in the array at all. If we compare using `> 1` we check if there are 3+ candidates, by changing it to `!== 0` we can check if there are 2+ candidates.

### CODE REVIEW

**Comment by rarkins** [comment]
Is it a logical error or just impacting the logger statement? And won't now it always log that if the files length was 1 to begin with?

File: lib/workers/pr/changelog/github/index.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export async function getReleaseNotesMd(
   }
   const { path: changelogFile, sha } = files.shift();
   /* istanbul ignore if */
-  if (files.length > 1) {
+  if (files.length !== 1) {
```
Comment (viceice) [inline lib/workers/pr/changelog/github/index.ts:73]:
```suggestion
  if (files.length !== 0) {
```

**Review by viceice — CHANGES_REQUESTED** [review]
Yes, the condition was wrong, we only log if more that two files are found. Now we log if more that one files is found

File: lib/workers/pr/changelog/github/index.ts (hunk 1 above)
Comment (Turbo87) [reply lib/workers/pr/changelog/github/index.ts]: whoops, thanks 😅

**Review by rarkins — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**